airlock/login_api.py

```diff
@@ -58,7 +56,38 @@ def get_user_data_dev(dev_users_file: Path, user: str, token: str):
     if user not in dev_users or dev_users[user]["token"] != token:
         raise LoginError("Invalid user or token")
     else:
-        details = dev_users[user]["details"]
-        # ensure that we never try refresh this user with job-server
-        details["last_refresh"] = time.time() + (365 * 24 * 60 * 60)
-        return details
+        return dev_users[user]["details"]
+
+
+def level4_auth_api_call(path, json):
```

Comment:
I have a not-very-strong objection to calling this `level4_...`; I know it reflects theview names in the jobserver endpoints that it's calling, but presumably at some point we might have Airlock in a non-TPP backend where it doesn't have a thing called level 4 that it's calling from. Is just `auth_api_call` too easilty confused with this current api? `external_auth_api_call`?

Reply:
So, I would expect that there would likely be an entirely different get_user_data/get_user_authz implementation in that case, and we would have a module level abstraction for it, and this function would be internal.

That said, level4_ is not a great name so I will changed it